content/applications/productivity/whatsapp.rst

```diff
@@ -0,0 +1,569 @@
+================
+WhatsApp in Odoo
+================
+
+WhatsApp is a messaging service operated by Meta. It is commonly used as a communication tool in
+many countries and within many businesses. This documentation will cover the integration of a
+WhatsApp Business Account with Odoo.
+
+There are two different setups that WhatsApp can have in Odoo. The first is for Odoo Online users
+(SaaS); this version works out-of-box and there are a set number of messages that can be sent and no
+messages can be received into the database from the customer. There is no setup required to use this
+version of WhatsApp with Odoo.
+
+The second version is that of a Meta business account, where the company's Meta account is
+configured in Odoo via a :abbr:`API (Application Programming Interface)` connection. This
+document will cover the second scenario.
```

Comment:
I'd focus on this default configuration, and  add the "Odoo Online" info as a short note afterwards.